lib/pure/collections/intsets.nim

```diff
@@ -253,35 +279,37 @@ proc incl*(s: var IntSet, other: IntSet) =
 
   for item in other: incl(s, item)
 
-proc toIntSet*(x: openArray[int]): IntSet {.since: (1, 3).} =
-  ## Creates a new IntSet that contains the elements of `x`.
+proc toOrdSet*[A](x: openArray[A]): OrdSet[A] {.since: (1, 3).} =
+  ## Creates a new OrdSet[A] that contains the elements of `x`.
   ##
   ## Duplicates are removed.
   ##
   ## See also:
-  ## * `initIntSet proc <#initIntSet>`_
+  ## * `initOrdSet[A] proc <#initOrdSet[A]>`_
   runnableExamples:
     var
-      a = toIntSet([5, 6, 7])
-      b = toIntSet(@[1, 8, 8, 8])
+      a = toOrdSet([5, 6, 7])
+      b = toOrdSet(@[1, 8, 8, 8])
     assert len(a) == 3
     assert len(b) == 2
 
-  result = initIntSet()
-  for item in items(x):
+  result = initOrdSet[A]()
+  for item in x:
     result.incl(item)
 
-proc containsOrIncl*(s: var IntSet, key: int): bool =
+proc toIntSet*(x: openArray[int]): IntSet {.since: (1, 3).} = toOrdSet[int](x)
```

Comment:
not `since`, `toIntSet` symbol already existed prior your PR (albeit with different implementation)

ditto wherever relevant

Reply:
I kept what it was in devel, toIntSet already has since 1.3